pkg/base1/test-dbus-address.js

```diff
@@ -3,6 +3,11 @@ import { common_dbus_tests, dbus_track_tests } from "./test-dbus-common.js";
 
 import QUnit from "qunit-tests";
 
+if (await QUnit.mock_info("bridge") == "cockpit-bridge.pyz")
```

Comment:
This is now spelled `await QUnit.mock_info("pybridge")`

Reply:
Changed.